userguide/conf.py

```diff
@@ -68,7 +68,7 @@
 }
 
 # Add any paths that contain custom themes here, relative to this directory.
-#html_theme_path = []
+html_theme_path = ['../themes']
```

Comment:
This looks like it's referring to a theme directory outside of the build directory (preprocessed).

I wonder if that themes directory should go in _static.  They are static files.

Otherwise, we can just have preproc copy the themes directory along with the other directories in the Makefile:

preproc:
        $(PREPROC) -t $(TAG) -o $(PREPROCDIR) -c 'conf.py _static artwork images themes'